internal/southboundhandler/handler.go

```diff
@@ -354,27 +374,16 @@ func extractBootstrapScript(secret *corev1.Secret, kind, providerID string) (str
 	if err != nil {
 		return "", err
 	}
-	switch {
-	case kind == "KubeadmConfig":
+	switch kind {
+	case configTypeKubeadm:
 		// Add providerID to Kubeadm node
-	case kind == "RKE2Config":
+	case configTypeKThrees:
+		dir := "/etc/rancher/k3s/config.yaml.d/"
```

Comment:
wonder if would be better set as constant. the same for rke2 path

Reply:
Yes, good catch.  Fixed.